sys-utils/hwclock.c

```diff
@@ -267,10 +276,10 @@ static int read_adjtime(const struct hwclock_control *ctl,
 
 	if (ctl->verbose) {
 		printf(_
-		       ("Last drift adjustment done at %ld seconds after 1969\n"),
-		       (long)adjtime_p->last_adj_time);
-		printf(_("Last calibration done at %ld seconds after 1969\n"),
-		       (long)adjtime_p->last_calib_time);
+		       ("Last drift adjustment done at %lld seconds after 1969\n"),
```

Comment:
_("Last drift adjustment done at %"PRIs64" seconds after 1969\n").

There is also typo () -> _()